content/source/docs/enterprise-beta/vcs/bitbucket-cloud.html.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: enterprise2
+page_title: "Bitbucket Cloud - VCS Services - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-vcs-bitbucket-cloud"
+---
+
+# Configuring Bitbucket Cloud Access
+
+These instructions are for using Bitbucket Cloud for Terraform Enterprise (TFE)'s VCS features. Bitbucket Cloud is the cloud-hosted version of Bitbucket; [self-hosted Bitbucket Server instances have separate instructions,](./bitbucket-server.html) as do the [other supported VCS services.](./index.html)
+
+Connecting TFE to your VCS involves five steps:
+
+On your VCS | On TFE
+--|--
+Register your TFE organization as a new app. Get ID and key. | &nbsp;
+&nbsp; | Tell TFE how to reach VCS, and provide ID and key. Get callback URL.
+Provide callback URL. | &nbsp;
+&nbsp; | Request VCS access.
+Approve access request. | &nbsp;
+
+The rest of this page explains the Bitbucket Cloud-specific versions of these steps.
+
+## Step 1: On Bitbucket Cloud, Create a New OAuth Consumer
+
+1. Open [Bitbucket Cloud](https://bitbucket.org) in your browser and log in as whichever account owns your organization's most important configuration.
```

Comment:
I'm talking about the account that does most of the administration for your org's VCS permissions. My (incomplete) understanding is that the default practice is to have a shared account for sort of ownership-level admin tasks, instead of assigning those permissions to personal accounts?